ladybug_charts/to_figure.py

```diff
@@ -62,7 +62,7 @@ def heat_map(
             of the data will be used. Defaults to None.
         show_title: A boolean to show or hide the title of the chart. Defaults to False.
         num_labels: The number of labels to be used in the legend. Defaults to None.
-        labels: A list of floats to be used as labels for the legend. Defaults to None.
+        labels: An ordinal_dictionary from ladybug legend parameters. Defaults to None.
```

Comment:
We should support both at this point. Otherwise, it will be a breaking change.

Reply:
agreed

Reply:
just sent another PR with a check for this